src/Plutus/Certification/API/Routes.hs

```diff
@@ -265,11 +262,34 @@ instance ToJSON DAppBody where
     , "githubToken"  .= dappGitHubToken
     ]
 
+newtype Twitter = Twitter { unTwitter :: Text }
+
+instance FromJSON Twitter where
+    parseJSON = withText "Twitter" $ \v ->
+      -- validate twitter account regex format ^[A-Za-z0-9_]{1,15}$
+      if not (isTwitterValid v)
+        then fail "Invalid twitter account format"
+        else pure $ Twitter v
+
+isTwitterValid :: Text -> Bool
+isTwitterValid v = not (
+  Text.length v > 15 || Text.length v < 1 || Text.any (\c -> not (isAlphaNum c || c == '_')) v
+  )
+
+instance ToJSON Twitter where
+  toJSON (Twitter v) = String v
+
+instance ToSchema Twitter where
+  declareNamedSchema _ = do
+    return $ NamedSchema (Just "Twitter") $ mempty
+      & type_ ?~ SwaggerString
+      & SL.pattern ?~ "^[A-Za-z0-9_]{1,15}$"
```

Comment:
Shouldn't we check if they added the "@" at the beginning? Something like: `^@?\w{1,15}$` ?

Reply:
For sure in the front, we should make such a constraint (@account_name) just to be sure the user understands, but for BE seems a bit redundant (the FE should remove the @ before calling the API).
I really don't have a strong opinion about this: your call

Reply:
Good point to have the filtering done by front-end.